### DISCUSSION TITLE
My SSL certificate is set to autorenew, but It received letsecrypt email about renew need

### DISCUSSION THREAD

**Discussion by sktbrd** [vercel/community#4651] [Feature Question] [answered]
### Question

<img width="870" alt="image" src="https://github.com/vercel/community/assets/116202536/c71da9d8-e7a3-4027-bd99-0c182e53fd0f">

<img width="870" alt="image" src="https://github.com/vercel/community/assets/116202536/9bcc792c-621b-4290-9a01-eff23d79f72a">

**Comment by swarnava** [comment, marked as answer]
Hi,

Your SSL certificate still has 18 days validity: https://www.sslshopper.com/ssl-checker.html#hostname=skatehive.app.

This email isn't generated by Vercel. I assume that someone must have used your email and domain to generate Letsencrypt certificate which is why you are getting email from Letsencrypt.

**Comment by sktbrd** [comment]
Thank you I will keep an eye on, maybe is the certbot I used in a VPS before using vercel?
